homeassistant/components/vicare/__init__.py

```diff
@@ -115,6 +116,31 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
+async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
I'm placing the comment here but it's related to `setup_entry`.
You need to register a listener so when the options are updated it reloads the integration. Ex: https://github.com/home-assistant/core/blob/dev/homeassistant/components/yale_smart_alarm/__init__.py#L25